packages/grafana-toolkit/src/cli/tasks/toolkit.build.ts

```diff
@@ -74,6 +76,27 @@ const moveFiles = () => {
   })();
 };
 
+const copySassFiles = () => {
+  const files = ['_variables.generated.scss', '_variables.dark.generated.scss', '_variables.light.generated.scss'];
+  // @ts-ignore
+  return useSpinner<void>(`Copy scss files ${files.join(', ')} files`, async () => {
+    const sassDir = path.resolve(cwd, '../../public/sass/');
+    const promises = files.map(file => {
+      return new Promise((resolve, reject) => {
+        fs.copyFile(`${sassDir}/${file}`, `${distDir}/sass/${file}`, err => {
```

Comment:
I think we could skip the `.generated.` from file name

Reply:
👍